integration/integration_test.go

```diff
@@ -6407,3 +6409,69 @@ func testListResourcesAcrossClusters(t *testing.T, suite *integrationTestSuite)
 		})
 	}
 }
+
+func testSFTP(t *testing.T, suite *integrationTestSuite) {
```

Comment:
Could any of this logic be written in in faster running unit tests as opposed to spinning up a whole integration test environment?

Reply:
Possibly, but I require a mostly-working Teleport instance to test SFTP due to the re-exec requirement so writing an integration test was much easier. Only takes a second or less on my machine, so it's not slow